package.json

```diff
@@ -81,7 +81,7 @@
     "nswatch": "^0.2.0",
     "politespace": "^0.1.4",
     "run-sequence": "^1.1.5",
-    "uswds": "1.2.0-rc1",
+    "uswds": "1.2.1",
```

Comment:
Is this change related to the USAJOBS case study, or was it just convenient to upgrade `uswds` in this PR?